src/domains/user/user.repository.ts

```diff
@@ -0,0 +1,42 @@
+import { DocumentData, DocumentReference } from 'firebase/firestore';
+import { IUser, IUserInfo, IUserUpdateInput } from './user.model';
```

Comment:
Please remove the prefix `I` from `IUser`, `IUserInfo`, `IUserUpdateInput`, and so on.